frontend/cypress/integration/editor/editor.spec.js

```diff
@@ -1,6 +1,17 @@
-describe('When on editor view, as a user', function () {
+describe('When visiting the application, as a user', function () {
```

Comment:
I think all these `function` keywords should be replaced with arrow functions to be more consistent with the rest of the application. Otherwise looks ok.

Reply:
Good point! Changed this now.